tests/test_mypy.py

```diff
@@ -1,61 +0,0 @@
-import importlib
```

Comment:
Moving this file both 1) makes it easier to dev the mypy tests since everything is closer together, and 2) causes the mypy tests to execute last. Given they are a little slow to execute, this seems preferable to me.